config/env.example

```diff
@@ -16,13 +16,15 @@
 ################################################################################################################################
 ################################################################################################################################
 
+# General
+LOG_LEVEL = TRACE
 
-# Serverless AWS config
-AWS_PROVIDER_PROFILE=default
-AWS_PROVIDER_REGION=local
-AWS_PROVIDER_STAGE=local
-AWS_SERVER_PORT=3009
+# Serverless AWS config for local
+AWS_SERVER_PORT=3020
 
-# DB config
-# ...
+SERVICE=trailers
 
+# DB config only for local
+DYNAMO_PORT = 8020
+DYNAMO_ENDPOINT = http://localhost:8020
```

Comment:
minor thing for us as this an example on how to document our env config via .env.
I assume our service does the following when retrieving the values 
```js
`${DYNAMO_ENDPOINT}:${DYNAMO_PORT}`
```
to concatenate the db endpoint